dist/gcc-compatible/EverCrypt_Hash.h

```diff
@@ -35,7 +35,9 @@ extern "C" {
 #include "krml/internal/target.h"
 
 
+#include "Lib_Memzero0.h"
```

Comment:
Why is this addition needed?

Reply:
Good question, I don't know

Reply:
I mean it should go into the .c file, or at the very least the internal header... I can investigate later on if it's important

Reply:
I think it's fine